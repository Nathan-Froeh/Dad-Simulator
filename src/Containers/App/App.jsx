import React, {Component} from 'react';
import './App.scss';

class App extends Component {

  render() {
  return (
    <div className="App">
    hi
    </div>
  );
  }
}

export default App;
