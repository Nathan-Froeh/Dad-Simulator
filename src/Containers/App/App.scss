
* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

html,
body,
#root,
.App {
  height: 100vh;
  width: 100vw;
}

.App {
  text-align: center;
  background: #EFC219;
}



