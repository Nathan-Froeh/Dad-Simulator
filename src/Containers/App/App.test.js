import React from 'react';
import ReactDOM from 'react-dom';
import {App} from './App';
import {shallow} from 'enzyme';


// 60 |      100 |       50 |       75 
// 55

describe('App', () => {
  let wrapper
  const props = {
    getFactOfTheDay: jest.fn()
  }
  beforeEach(() => {
    wrapper = shallow(<App {...props}/>)

  })

  it('App should mathc snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('componentDidMount should call getFactOfTheDay', () => {
    expect(props.getFactOfTheDay).toHaveBeenCalledTimes(2)
    wrapper.instance().componentDidMount()
    expect(props.getFactOfTheDay).toHaveBeenCalledTimes(3)
  })

})
