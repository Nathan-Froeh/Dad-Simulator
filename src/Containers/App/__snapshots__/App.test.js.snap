// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App should mathc snapshot 1`] = `ShallowWrapper {}`;
